anyway/widgets/urban_widgets/urban_crosswalk_widget.py

```diff
@@ -56,20 +54,22 @@ def get_crosswalk(yishuv, street, start_time, end_time, resolution) -> Dict[str,
                         InjurySeverity.SEVERE_INJURED.value,
                     ],
                     "cross_location": CrossCategory.NONE.get_codes(),
-                    "accident_yishuv_name": yishuv,
-                    "street1_hebrew": street,
+                    "accident_yishuv_symbol": location_info["yishuv_symbol"],
+                    "street1": location_info["street"],
                 },
-                group_by="street1_hebrew",
-                count="street1_hebrew",
+                group_by="street1",
+                count="street1",
                 start_time=start_time,
                 end_time=end_time,
                 resolution=resolution,
             ),
         }
         if not cross_output["with_crosswalk"]:
-            cross_output["with_crosswalk"] = [{"street1_hebrew": street, "count": 0}]
+            cross_output["with_crosswalk"] = [{"street1_hebrew": location_info["_hebrew"],
+                                               "count": 0}]
         if not cross_output["without_crosswalk"]:
-            cross_output["without_crosswalk"] = [{"street1_hebrew": street, "count": 0}]
+            cross_output["without_crosswalk"] = [{"street1_hebrew": location_info["street_hebrew"],
+                                                   "count": 0}]

```

Comment:
is it `location_info["street_hebrew"]` or should it be location_info["street1_hebrew"]?